crates/gitbutler-core/src/virtual_branches/virtual.rs

```diff
@@ -138,7 +138,7 @@ pub struct VirtualBranchHunk {
     pub id: String,
     #[serde(serialize_with = "crate::serde::as_string_lossy")]
     pub diff: BString,
-    pub modified_at: u128,
+    pub modified_at: Duration,
```

Comment:
Maybe `Duration` is convenient because it naturally translates into a number, maybe seconds with sub-second precision?
Nonetheless I think the Rust-side should keep what makes sense and is lossless, and then transform during serialization.